spec/controller/application_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe ApplicationController, type: :controller do
+  describe "#request_ip" do
+    it "returns the anonymized IP with the last octet zero padded" do
+      expect(controller.request_ip).to eql("0.0.0.0")
```

Comment:
I'm struggling with this, I can't work out the setup? Does an RSpec controller test (contraversial! :P) have a preset IP? Has this done anything to it and how would I know?

Reply:
I've made this clearer by explicitly passing a v4 IP address